controllers/subscriptions.go

```diff
@@ -31,7 +31,7 @@ var cache = ccache.New(
 		MaxSize(configOptions.GetInt64(config.Keys.SubsCacheMaxSize)).
 		ItemsToPrune(configOptions.GetUint32(config.Keys.SubsCacheItemPrune)),
 )
-var cacheDurationSeconds = time.Second * time.Duration(configOptions.GetInt64(config.Keys.SubsCacheDuration))
+var cacheDuration = time.Second * time.Duration(configOptions.GetInt64(config.Keys.SubsCacheDuration))
```

Comment:
unrelated, I got a suggestion from `go-staticcheck` linter to remove `Seconds` from the var name so I did it